### PR TITLE
Add regression test for `System.Windows.Forms` bug

### PR DESCRIPTION
**Pull request by andschwa**
This PR will fail until we can update OmniSharp and so include https://github.com/OmniSharp/csharp-language-server-protocol/pull/633, which fixes the bug this regression test covers. When that happens, this PR will fix:

* https://github.com/PowerShell/vscode-powershell/issues/3410
* https://github.com/PowerShell/vscode-powershell/issues/3396
* https://github.com/PowerShell/vscode-powershell/issues/3394 (minus a separate issue we're still working out)
* https://github.com/PowerShell/vscode-powershell/issues/3415 (probably anyway, still waiting to hear back from the original author)

### CODE REVIEW

**Comment by andschwa** [comment]
Whelp, it's _passing_, which isn't right!

**Comment by andschwa** [comment]
Ok, I need to actually write this test in the style of:

https://github.com/PowerShell/PowerShellEditorServices/blob/00c78242e17a7265706a0fa07b67770709a670ac/test/PowerShellEditorServices.Test.E2E/DebugAdapterProtocolMessageTests.cs#L168

Because it's the OmniSharp integration that is the regression to assert.

**Comment by andschwa** [comment]
Ok, @rjmholt that build failure _looks_ like the expected hang to me, I only wish I could see that locally.

**Comment by andschwa** [comment]
Look at that! Regression test hung before the update, and doesn't after!!!

File: test/PowerShellEditorServices.Test/PowerShellEditorServices.Test.csproj [hunk 1]
```diff
@@ -34,8 +34,7 @@
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
     <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
-    <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="0.19.2" />
-    <DotNetCliToolReference Include="dotnet-xunit" Version="2.4.0-beta.1.build3958" />
```
Comment (andschwa) [inline test/PowerShellEditorServices.Test/PowerShellEditorServices.Test.csproj:38]: Not sure what on earth this `DotNetCliToolReference` was for, it doesn't seem needed any more.

**Review by rjmholt — APPROVED** [review]

**Comment by andschwa** [comment]
@rjmholt I dropped this back to OmniSharp v0.19.3 as it's all that's on NuGet. v0.19.4 will come eventually, we'll just let Dependabot update it then.

**Pull request merged**